src/OpenIddict.Security/OpenIddictExtensions.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Microsoft.Extensions.Internal;
+using NWebsec.Middleware;
+using System;
+
+namespace Microsoft.AspNet.Builder {
+    public static class OpenIddictExtensions {
+        public static OpenIddictBuilder UseSecurityHeaders([NotNull] this OpenIddictBuilder builder, Action<IFluentCspOptions> cspOptions = null) {
```

Comment:
As you mentioned, it's closely tied to NWebsec, so `UseNWebsec` is a good name :smile: